sdk/textanalytics/Azure.AI.TextAnalytics/tests/samples/Sample_HealthcareBatchConvenienceAsync.cs

```diff
@@ -22,48 +22,44 @@ public async Task HealthcareBatchConvenienceAsync()
+            string document = "Subject is taking 100mg of ibuprofen twice daily";
 
-            List<string> batchInput = new List<string>()
+            var list = new List<string>();
+
+            for (int i = 0; i < 6; i++)
             {
-                document,
+                list.Add(document);
             };
 
-            AnalyzeHealthcareEntitiesOperation healthOperation = await client.StartAnalyzeHealthcareEntitiesAsync(batchInput, "en");
+            AnalyzeHealthcareEntitiesOperation healthOperation = await client.StartAnalyzeHealthcareEntitiesAsync(list, "en", new AnalyzeHealthcareEntitiesOptions() { IncludeStatistics = true } );
 
             await healthOperation.WaitForCompletionAsync();
 
-            AnalyzeHealthcareEntitiesResultCollection results = healthOperation.Value;
-
-            Console.WriteLine($"Results of Azure Text Analytics \"Healthcare Async\" Model, version: \"{results.ModelVersion}\"");
-            Console.WriteLine("");
-
-            foreach (AnalyzeHealthcareEntitiesResult result in results)
+            await foreach (AnalyzeHealthcareEntitiesResultCollection documentsInPage in healthOperation.Value)
             {
-                Console.WriteLine($"    Recognized the following {result.Entities.Count} healthcare entities:");
+                Console.WriteLine($"Results of Azure Text Analytics \"Healthcare Async\" Model, version: \"{documentsInPage.ModelVersion}\"");
+                Console.WriteLine("");
 
-                foreach (HealthcareEntity entity in result.Entities)
+                foreach (AnalyzeHealthcareEntitiesResult entitiesInDoc in documentsInPage)
                 {
-                    Console.WriteLine($"    Entity: {entity.Text}");
-                    Console.WriteLine($"    Category: {entity.Category}");
-                    Console.WriteLine($"    Offset: {entity.Offset}");
-                    Console.WriteLine($"    Length: {entity.Length}");
-                    Console.WriteLine($"    Links:");
-
-                    foreach (EntityDataSource entityDataSource in entity.DataSources)
+                    if (!entitiesInDoc.HasError)
                     {
-                        Console.WriteLine($"        Entity ID in Data Source: {entityDataSource.EntityId}");
-                        Console.WriteLine($"        DataSource: {entityDataSource.Name}");
+                        foreach (var entity in entitiesInDoc.Entities)
+                        {
+                            Console.WriteLine($"    Entity: {entity.Text}");
+                            Console.WriteLine($"    Category: {entity.Category}");
+                            Console.WriteLine($"    Offset: {entity.Offset}");
+                            Console.WriteLine($"    Length: {entity.Length}");
+                            Console.WriteLine($"    Links:");
```

Comment:
The links for-loop is misssing here

Reply:
thank you! I deleted for internal testing and forgot to add it. I will bring it back